### ISSUE TITLE
WARNING:maft.data.datasets.load_sem_seg:Directory coco/train2017 and coco/stuffthingmaps_detectron2/train2017 has 118287 and 118280 files, respectively.

### ISSUE THREAD

**Issue by freeLGR** [hustvl/MaskAdapter#6]
Hello!Thank you very much for your outstanding work!
I encountered a problem. I found that using the python script you provided to generate the coco training data set, there was a problem. The number of generated labels is inconsistent with the number of images in the training set. How did you solve it?

The detailed log is as follows:
WARNING:maft.data.datasets.load_sem_seg:Directory coco/train2017 and coco/stuffthingmaps_detectron2/train2017 has 118287 and 118280 files, respectively.

Moreover, I found that the 7 pictures without generated labels are all pure white pictures.

**Comment by owl-10** [comment]
We obtained the scripts for processing COCO-Stuff from MAFT [here](https://github.com/jiaosiyu1999/MAFT/blob/master/datasets/prepare_coco_stuff_sem_seg.py) ,and we also encountered the same warning. It seems to be an issue within the MAFT script. You might want to consider opening an issue on their repository. However, this issue doesn’t seem to have a significant impact on the results.

**Comment by freeLGR** [comment]
Thank you for your answer!
